measureCpu.py

```diff
@@ -42,7 +45,28 @@ def stop(self):
         data = jc.parse('mpstat', r.out)
         p_idle = data[0]['percent_idle']
         logger.info(f"Idle on {self.node_name} = {p_idle}%")
+        self._output = self.generate_output(data)
 
-    def output(self):
-        #TODO: handle printing/storing output here
-        pass
+    def output(self, out: TftAggregateOutput):
+        # Return machine-readable output to top level
+        if self.is_client:
+            out.measureCPU_client_output = self._output
+        else:
+            out.measureCPU_server_output = self._output
+
+        # Print summary to console logs
+        p_idle = self._output.result['percent_idle']
+        logger.info(f"Idle on {self.node_name} = {p_idle}%")
+
+    #TODO: We are currently only storing the "cpu: all" data from mpstat
+    def generate_output(self, data) -> PluginOutput:
+        return PluginOutput(
+            plugin_metadata={
+                "name": "MeasureCPU",
+                "node_name": self.node_name,
+                "pod_name": self.pod_name,
+            },
+            command=self.cmd,
+            result=data[0],
```

Comment:
Do we want this entire output or could we parse it to be something more useful?

Reply:
We definitely want to parse it eventually, however this is an early implementation of measureCPU. Eventually we will need to synchronize the intervals of each measurement with the flow test results and generate some sort of machine readable object. For the time being I opted to just put this as a placeholder for something more sophisticated.